### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by AA666666**

### CODE REVIEW

File: client/src/listings/Listings.js [hunk 1]
```diff
@@ -41,28 +48,40 @@ function Listings(props) {
   const [subcategoryId, setSubcategoryId] = useState('');
   const [categoryOptions, setCategoryOptions] = useState([]);
   const [subcategoryOptions, setSubcategoryOptions] = useState({});
+  const [page, setPage] = useState(1);
+  const [pageSize, setPageSize] = useState(15);
```
Comment (JadenBalogh) [inline client/src/listings/Listings.js:52]: This should not be a state variable, it should just be a constant. Its value never changes.

File: client/src/listings/Listings.js [hunk 2]
```diff
@@ -41,28 +48,40 @@ function Listings(props) {
...
+  const [pageNum, setPageNum] = useState(0);
```
Comment (JadenBalogh) [inline client/src/listings/Listings.js:53]: This should be renamed to a more clear variable name, maybe `totalPages` or `pageCount`.

File: client/src/listings/Listings.js [hunk 3]
```diff
@@ -95,6 +114,8 @@ function Listings(props) {
 
   function handleCategoryChange(event) {
     setCategoryId(event.target.value);
+    console.log('suboptions', subcategoryOptions);
+    console.log('cateId', categoryId);
```
Comment (JadenBalogh) [inline client/src/listings/Listings.js:118]: It's generally best to remove extraneous console logs in finished code.

File: client/src/profile/ProfileField.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+          <Grid className={classes.divider} item xs={9} align='left'>
+            <Typography variant='body2'>{`${details}`}</Typography>
+          </Grid>
+        </Grid>
+      ) : (
+        <></>
+      )}
+    </Grid>
+  );
+}
```
Comment (JadenBalogh) [inline client/src/profile/ProfileField.js:33]: Why was this added? This appears to be an exact copy of the shared ProfileField component, and is not being used anywhere that I can tell.

**Review by JadenBalogh — CHANGES_REQUESTED** [review]
The pagination looks good at a glance - have you tested it with multiple pages? I made a minor change to update the formatting and reformat the server code to match our standards and remove some extraneous variables. Overall, seems to work well!

I have made some comments for minor changes to add below.

**Review by aKrisnanto — COMMENTED** [review]
Doesn't seem to be working on my end

**Pull request closed without merging**